backend/openapi.yaml

```diff
@@ -4734,6 +4733,20 @@ components:
         status:
           type: string
           enum: [UNSENT, ACCEPTED, SENT, BOUNCED, DELIVERED, OPENED, COMPLAINT]
+          description: >
+            * `UNSENT` - initial state of a newly created EmailMessageTransactional
+
+            * `ACCEPTED` - email has been accepted by our email provider; this state is returned upon a successful API request
+
+            * `SENT` - the send request to our email provider was successful and our email provider will attempt to deliver the message to the recipient’s mail server
+
+            * `BOUNCED` - the recipient's mail server rejected the email
+
+            * `DELIVERED` - the email provider has successfully delivered the email to the recipient's mail server
+
+            * `OPENED` - the recipient received the message and opened it in their email client
+
+            * `COMPLAINT` - the email was successfully delivered to the recipient’s mail server, but the recipient marked it as spam
```

Comment:
the relevant change is here!

Reply:
so some weirdness I observed:
- on Swagger's [official website](https://swagger.io/docs/specification/data-models/enums/#:~:text=All%20values%20in%20an%20enum%20must%20adhere%20to%20the%20specified%20type.%20If%20you%20need%20to%20specify%20descriptions%20for%20enum%20items%2C%20you%20can%20do%20this%20in%20the%20description%20of%20the%20parameter%20or%20property%3A), they did not leave line breaks after each line of the enum description
- but if I didn't do that, the enum description is all clustered as a single sentence

<img width="787" alt="image" src="https://user-images.githubusercontent.com/67887489/210127921-043cbc04-207f-43ee-bfc3-073ab8a0e6f5.png">

![image](https://user-images.githubusercontent.com/67887489/210127933-d892f7a8-1a2e-4090-9fad-7244bb565170.png)